src/gstreamer/gstkvssink.h

```diff
@@ -70,111 +65,97 @@ typedef struct _KvsSinkMetric KvsSinkMetric;
+    GstCollectData collect; /* we extend the CollectData */
     MKV_TRACK_INFO_TYPE track_type;
-    GstKvsSink *kvssink;
+    GstKvsSink* kvssink;
     guint track_id;
 } GstKvsSinkTrackData;
 
-typedef enum _MediaType {
-    AUDIO_VIDEO,
-    VIDEO_ONLY,
-    AUDIO_ONLY
-} MediaType;
+typedef enum _MediaType { AUDIO_VIDEO, VIDEO_ONLY, AUDIO_ONLY } MediaType;
 
 /**
  * GstKvsSink:
  *
  * The opaque #GstKvsSink data structure.
  */
 struct _GstKvsSink {
-    GstElement		element;
+    GstElement element;
 
-    GstCollectPads              *collect;
+    GstCollectPads* collect;
 
     // Stream definition
-    gchar                       *stream_name;
-    gchar                       *user_agent;
-    guint                       retention_period_hours;
-    gchar                       *kms_key_id;
-    STREAMING_TYPE              streaming_type;
-    gchar                       *content_type;
-    guint                       max_latency_seconds;
-    guint                       fragment_duration_miliseconds;
-    guint                       timecode_scale_milliseconds;
-    gboolean                    key_frame_fragmentation;
-    gboolean                    frame_timecodes;
-    gboolean                    absolute_fragment_times;
-    gboolean                    fragment_acks;
-    gboolean                    restart_on_error;
-    gboolean                    recalculate_metrics;
-    gboolean                    allow_create_stream;
-    gboolean                    disable_buffer_clipping;
-    guint                       framerate;
-    guint                       avg_bandwidth_bps;
-    guint                       buffer_duration_seconds;
-    guint                       replay_duration_seconds;
-    guint                       connection_staleness_seconds;
-    gchar                       *codec_id;
-    gchar                       *track_name;
-    gchar                       *access_key;
-    gchar                       *secret_key;
-    gchar                       *session_token;
-    gchar                       *aws_region;
-    guint                       rotation_period;
-    gchar                       *log_config_path;
-    guint                       storage_size;
-    guint                       stop_stream_timeout;
-    guint                       service_connection_timeout;
-    guint                       service_completion_timeout;
-    gchar                       *credential_file_path;
-    GstStructure                *iot_certificate;
-    GstStructure                *stream_tags;
-    guint64                     file_start_time;
-    MKV_TRACK_INFO_TYPE         track_info_type;
-    gchar                       *audio_codec_id;
-
-
-    guint                       num_streams;
-    guint                       num_audio_streams;
-    guint                       num_video_streams;
-
+    gchar* stream_name;
+    gchar* user_agent;
+    guint retention_period_hours;
+    gchar* kms_key_id;
+    STREAMING_TYPE streaming_type;
```

Comment:
Is there a way to keep it how it was before, I think the before version here is clearer. We should stick to one way and be consistent, the consistency is most important.

Reply:
In Producer C, we do use this same style of not having indentations to align variable names - there are indentations for "#define"s though. I tested and searched around to find that the way clang-format works is it parses all the code and then puts it back together according to the configured format, so there is no way to remove this formatting. There may be a formatting rule that would add this spacing between variable type and name, but it would then apply to _all_ variables. The two solutions I see are to either use another formatter, or place the following around code we do not want to format:
```
// clang-format off
...
// clang-format on
```